src/references/strings.md

```diff
@@ -11,7 +11,7 @@ Including `&str` as a way of representing a slice of valid utf-8
 We can now understand the two string types in Rust:
 
 - `&str` is a slice of UTF-8 encoded bytes, similar to `&[u8]`.
-- `String` is an owned, heap-allocated buffer of UTF-8 bytes.
+- `String` has ownership over its heap-allocated buffer of UTF-8 encoded bytes.
```

Comment:
Now that I see the two bullet points together, I feel it would be good for them to have a similar structure: "Foo _is_ X, similar to Y".

So what about this:

```suggestion
- `String` is an owned buffer of UTF-8 encoded bytes, similar to `Vec<T>`.
```

Reply:
It is ok for me but I want to make a suggestion. Then the desired one can be selected.

```suggestion
- `String` is a type that has ownership over its heap-allocated buffer of UTF-8 encoded bytes.
```

Reply:
Since we say "the two string types in Rust" just above, I think adding "is a type" here isn't necessary?

Reply:
Yes you are right. I missed this.